playwright/test/works-search.test.ts

```diff
@@ -65,11 +65,7 @@ const navigateToNextPage = async (page: Page) => {
   // another hack
   await Promise.all([
     safeWaitForNavigation(page),
-    page.click(
-      `[id="pagination"]${
-        isMobile(page) ? ':not(.is-hidden-s)' : ''
-      }:nth-of-type(1) a`
-    ),
+    page.click(`[id="pagination"]:nth-of-type(1) button`),
```

Comment:
Does this imply we have multiple elements with the `id` pagination? Or have I misunderstood?

Reply:
Yes - which was wrong! We have two pagination elements in each page - the one at the top is hidden on mobile and the one at the bottom stays. The changes I made would set them apart, but you're right that I can remove `nth-of-type(1)`, which was legacy. I'll do that now!

Reply:
Actually I'll also change it to use `data-test-id` which makes its use clearer and is used across the codebase.

Reply:
My concern is less about the selector, more that two elements on the page might have the same `id`, which feels wrong to me, but maybe I've missed something?

Reply:
Yes, it was 100% wrong 👍 using `data-test-id` will also fix that.